UnitTests/BraintreeShopperInsightsTests/BTShopperInsightsClient_Tests.swift

```diff
@@ -41,6 +43,37 @@ class BTShopperInsightsClient_Tests: XCTestCase {
         XCTAssertTrue(result!.isVenmoRecommended)
     }
     
+    func testGetRecommendedPaymentMethods_whenAppsNotInstalled_callsEligiblePaymentsAPI() async {
+        _ = try? await sut.getRecommendedPaymentMethods(request: request)
+        
+        XCTAssertEqual(mockAPIClient.lastPOSTPath, "/v2/payments/find-eligible-methods")
+        XCTAssertEqual(mockAPIClient.lastPOSTAPIClientHTTPType, .paypalAPI)
+        
+        guard let lastPostParameters = mockAPIClient.lastPOSTParameters else {
+            XCTFail()
+            return
+        }
+        
+        let customer = lastPostParameters["customer"] as! [String: Any]
```

Comment:
🏈 Would love for us to implement a cleaner way to verify JSON structure across our SDKs test suite

Reply:
On Android, we use https://github.com/skyscreamer/JSONassert.
Maybe you could try using https://github.com/iwill/generic-json-swift on iOS?

Reply:
👀 Thanks for those resources! Instead of doing it in this PR, I was thinking we could explore some options and once we land on one we like update all of our JSON related tests to use it. I can throw this in the backlog